src/sentry/api/endpoints/auth_login.py

```diff
@@ -0,0 +1,97 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.app import ratelimiter
+from sentry.utils import auth, metrics
+from sentry.utils.hashlib import md5_text
+from sentry.api.base import Endpoint
+from sentry.api.serializers.base import serialize
+from sentry.api.serializers.models.user import DetailedUserSerializer
+from sentry.web.forms.accounts import AuthenticationForm
+from sentry.web.frontend.base import OrganizationMixin
+
+
+class AuthLoginEndpoint(Endpoint, OrganizationMixin):
+    # Disable authentication and permission requirements.
+    permission_classes = []
+
+    def org_redirect_url(self, request):
```

Comment:
This is copy/pasted from `AuthConfigEndpoint.org_redirect_url`, can you DRY this up?